pkg/webhook/controlplane/genericmutator/mutator.go

```diff
@@ -92,6 +97,17 @@ func (m *mutator) Mutate(ctx context.Context, obj runtime.Object) error {
 		case common.KubeSchedulerDeploymentName:
 			return m.ensurer.EnsureKubeSchedulerDeployment(ctx, x)
 		}
+	case *appsv1.StatefulSet:
+		switch x.Name {
+		case common.EtcdMainStatefulSetName, common.EtcdEventsStatefulSetName:
```

Comment:
Maybe I missed this part: How do we ensure that the `etcd-events` stateful set does get the backup-restore sidecar container, however, configured *WITHOUT* backup?

Reply:
As discussed, it's me who missed this. Now it should be ok, changed it in `controlplane.GetBackupRestoreContainer`.